icechunk-python/src/lib.rs

```diff
@@ -122,6 +120,31 @@ impl From<SnapshotMetadata> for PySnapshotMetadata {
     }
 }
 
+#[derive(FromPyObject)]
+enum ChecksumArgument {
```

Comment:
@mpiannucci is this the right approach to take a python argument that can be any of these types?

Reply:
Yes!